src/Subscriptions/SubscriptionBroadcaster.php

```diff
@@ -100,11 +106,11 @@ function (Subscriber $subscriber) use ($root) {
     /**
      * Authorize the subscription.
      *
-     * @param Request $request
+     * @param  \Illuminate\Http\Request  $request
      *
-     * @return array
+     * @return \Illuminate\Http\Response
      */
-    public function authorize(Request $request)
+    public function authorize(Request $request): Response
```

Comment:
This can also be a `Illuminate\Http\JsonResponse`

Reply:
How about responses in general? What should the return type in the controller and in `GraphQLResponse` be?

Reply:
`GraphQLResponse` is an interface which can be extended by the user so no return type goes there. As for the Controller, although users can technically return an array if they prefer, Laravel uses `\Illuminate\Http\Response` in its docblocks w/out an enforced return type so let's keep it the same.

Reply:
Since we do use the `StreamedResponse`, how about we annotate it as `\Symfony\Component\HttpFoundation\StreamedResponse`. All response types i have identified are actually subclasses of that.

Reply:
👍 That works for me. Pipe this through some local testing and get it merged in if everything is good (should be this afternoon). Thanks @spawnia!!

Reply:
Actually i meant to say to change all hints to `\Symfony\Component\HttpFoundation\Response`, the last commit does exactly that. I think you got that, though :)